### PR TITLE
extract deployment configurations to config file

### PR DESCRIPTION
**Pull request by peishanwang**

### CODE REVIEW

File: modules/application/src/main/java/edu/northeastern/truthtree/ApplicationConfig.java [hunk 1]
```diff
@@ -1,118 +1,24 @@
...
-
-  @Bean
-  public ISimilarLocationsAdapter getSimilarLocationsAdapter() {
-    return RETURN_MOCK_DATA_SUPPORTED_ATTRIBUTES ?
-            new SimilarLocationsMockAdapter() :
-            new SimilarLocationsDBAdapter(new SimilarLocationsAssembler(new ObjectMapper()));
-
-  }
+  @Value("${elasticsearchUrl}")
+  private String elasticsearch_endpoint;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/ApplicationConfig.java]: This should probably be in `camelCase`

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java [hunk 2]
```diff
@@ -18,14 +14,22 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.UriComponentsBuilder;
 
 @Component("similarLocationsDBAdapter")
 public class SimilarLocationsDBAdapter implements ISimilarLocationsAdapter {
 
+  @Value("${databaseUrl}")
+  private String db_endpoint;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java]: `camelCase` naming?

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java [hunk 3]
```diff
@@ -18,14 +14,22 @@
...
+
+  @Value("${mlEndpoint}")
+  private String ml_endpoint;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java]: `camelCase` naming?

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/attributes/AttributesDBAdapter.java [hunk 4]
```diff
@@ -1,30 +1,36 @@
...
 import org.json.simple.JSONArray;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 import org.springframework.web.util.UriComponentsBuilder;
 
+@Component
 public class AttributesDBAdapter implements IAttributesAdapter {
 
+  @Value("${databaseUrl}")
+  private String db_endpoint;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/adapter/attributes/AttributesDBAdapter.java]: `camelCase` naming?

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/collections/CollectionsDBAdapter.java [hunk 5]
```diff
@@ -7,19 +7,25 @@
...
 import org.json.simple.JSONArray;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 import org.springframework.web.util.UriComponentsBuilder;
 
+@Component
 public class CollectionsDBAdapter implements ICollectionsAdapter {
 
+  @Value("${databaseUrl}")
+  private String db_endpoint;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/adapter/collections/CollectionsDBAdapter.java]: `camelCase` naming?

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/timerange/TimeRangeDBAdapter.java [hunk 6]
```diff
@@ -3,13 +3,18 @@
...
+import org.springframework.stereotype.Component;
 import org.springframework.web.util.UriComponentsBuilder;
 
-import java.util.List;
-
+@Component
 public class TimeRangeDBAdapter implements ITimeRangeAdapter {
 
+  @Value("${databaseUrl}")
+  private String db_endpoint;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/adapter/timerange/TimeRangeDBAdapter.java]: `camelCase` naming?

File: modules/application/src/main/java/edu/northeastern/truthtree/service/login/LoginService.java [hunk 7]
```diff
@@ -1,15 +1,18 @@
...
 
 import org.mindrot.jbcrypt.BCrypt;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Component
 public class LoginService implements ILoginService {
 
+  @Value("${passwordHash}")
+  private String password_hash;
```
Comment (steventhan) [inline modules/application/src/main/java/edu/northeastern/truthtree/service/login/LoginService.java]: passwordHash

File: modules/application/src/main/java/edu/northeastern/truthtree/ApplicationConfig.java (hunk 1 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/ApplicationConfig.java]: ack

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java (hunk 3 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java]: ack

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/attributes/AttributesDBAdapter.java (hunk 4 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/adapter/attributes/AttributesDBAdapter.java]: ack

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java (hunk 2 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/adapter/advancedsearch/SimilarLocationsDBAdapter.java]: ack

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/collections/CollectionsDBAdapter.java (hunk 5 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/adapter/collections/CollectionsDBAdapter.java]: ack

File: modules/application/src/main/java/edu/northeastern/truthtree/adapter/timerange/TimeRangeDBAdapter.java (hunk 6 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/adapter/timerange/TimeRangeDBAdapter.java]: ack

File: modules/application/src/main/java/edu/northeastern/truthtree/service/login/LoginService.java (hunk 7 above)
Comment (peishanwang) [reply modules/application/src/main/java/edu/northeastern/truthtree/service/login/LoginService.java]: ack

**Pull request merged**